streams/src/test/java/org/apache/kafka/streams/kstream/internals/MaterializedInternalTest.java

```diff
@@ -73,11 +71,11 @@ public void shouldUseProvidedStoreNameWhenSet() {
     @Test
     public void shouldUseStoreNameOfSupplierWhenProvided() {
         final String storeName = "other-store-name";
-        EasyMock.expect(supplier.name()).andReturn(storeName).anyTimes();
-        EasyMock.replay(supplier);
+        when(supplier.name()).thenReturn(storeName);
         final MaterializedInternal<Object, Object, KeyValueStore<Bytes, byte[]>> materialized =
             new MaterializedInternal<>(Materialized.as(supplier), nameProvider, prefix);
         assertThat(materialized.storeName(), equalTo(storeName));
+        verify(supplier, times(2)).name();
```

Comment:
Why `times(2)`? The original test does not says anything about the number of calls. I think if you swap line 77 with line 78, you can remove `times(2)`.

Reply:
That's a fair point, swapped lines 77 with 78 and removed `times(2)` in upcoming commits.